src/ts/component/SearchFilters/SearchFilters.tsx

```diff
@@ -0,0 +1,222 @@
+import Form from 'react-bootstrap/Form'
+import SearchFilterSection from '../SearchFilterSection/SearchFilterSection'
+import SearchFilterSectionList from '../SearchFilterSectionList/SearchFilterSectionList'
+import FacetChecklist from '../FacetChecklist/FacetChecklist'
+
+import { Facet } from '../../../types/global'
+
+import './SearchFilters.scss'
+
+interface SearchFiltersProps {
+  facets: object;
+  filterValues: {
+    bounding_box?: string
+    processing_level_id_h?: string[];
+    science_keywords_h?: string[];
+    temporal?: string[] | string
+    page_num?: number
+  };
+  handleChange: (_event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleBlur: (_event: React.FocusEvent<HTMLInputElement>) => void;
+  setQueryString: (_query: string) => void;
+  setSidebarOpened: (_isOpened: boolean) => void;
+}
+
+const findFacet = (facets: Facet, title: string): Facet | null => {
+  if (!facets || !facets.children) return null
+
+  return facets.children.find((c) => c.title === title) || null
+}
+
+const findChildFacets = (facets: Facet, title: string): Facet[] => {
+  const facet = findFacet(facets, title)
+  if (!facet || !facet.children) return []
+
+  return facet.children
+}
+
+const ickyValues = ['-Na-', 'Not Provided', 'Other (Ames)']
+
+const namesToParams: { [key: string]: string } = {
+  Keywords: 'science_keywords_h',
+  Platforms: 'platforms_h',
+  'Horizontal Data Resolution': 'horizontal_data_resolution_range',
+  'Data Format': 'granule_data_format_h',
+  'Processing Levels': 'processing_level_id'
+}
+
+const SearchFilters: React.FC<SearchFiltersProps> = ({
+  facets,
+  filterValues,
+  handleChange,
+  handleBlur,
+  setQueryString,
+  setSidebarOpened
+}) => {
+  const onChange = (facet: Facet, isChecked: boolean) => {
+    const { apply, remove } = facet.links || {}
+    if (isChecked && apply) {
+      setQueryString(apply.split('?')[1])
+    }
+
+    if (!isChecked && remove) {
+      setQueryString(remove.split('?')[1])
+    }
+  }
+
+  const getFacets = (t: string) => findChildFacets(facets as Facet, t)
+    .filter(({ title }) => !ickyValues.includes(title))
+
+  return (
+    <SearchFilterSectionList
+      defaultActiveKey={['1']}
+      setSidebarOpened={setSidebarOpened}
+    >
+      {/* Unimplemented. In Data Catalog comps but not site search comps. TBD if needed.
+      <Accordion.Item eventKey="0">
+        <Accordion.Header>Filter By Text</Accordion.Header>
+        <Accordion.Body />
+      </Accordion.Item> */}
+      {
+        getFacets('Keywords').length > 0 && (
+          <SearchFilterSection title="Topics" eventKey="1" setSidebarOpened={setSidebarOpened}>
+            <FacetChecklist
+              name="topics"
+              param={namesToParams.Keywords}
+              facets={getFacets('Keywords')}
+              onChange={onChange}
+            />
+          </SearchFilterSection>
+        )
+      }
+      {/* Commented out until section is implemented
```

Comment:
Do we have a ticket to implement this? If so, add the ticket number to the comment